lib/puppet-lint/checkplugin.rb

```diff
@@ -59,6 +59,14 @@ def tokens
     PuppetLint::Data.tokens
   end
 
+  def add_token(index, token)
```

Comment:
Should this have inputs relative to the problem token, rather than a specific index?